packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -52,9 +52,13 @@ export class EpochTracker implements IPersistedFileCache {
         protected readonly cache: IPersistedCache,
         protected readonly fileEntry: IFileEntry,
         protected readonly logger: ITelemetryLogger,
+        protected readonly cacheExpiryTimeoutOverrideMs: number | undefined,
```

Comment:
You can make this optional. Then you don't need to update it places where you don't want to pass in a value.

Reply:
Basically, you don't need to update most of the tests.

Reply:
I thought about doing so, but I don't want someone to create an EpochTracker without knowing about this override value. It's rather important to pass in if it's defined.